### ISSUE TITLE
Disclaimers should be displayed together with results from each database

### ISSUE THREAD

**Issue by allasm** [MatchMakerExchange/mme-apis#38]
...though this is probably not part of the API itself

**Comment by Relequestual** [comment]
I agree. This should be in some documented policy.

**Comment by Relequestual** [comment]
@MatchMakerExchange/committers Another PING! =]

**Comment by buske** [comment]
We should have the PhenomeCentral one approved within ~12 hours. It sounds like it might be eerily similar to the Decipher one. :)

**Comment by fschiettecatte** [comment]
I have reminded Ada of this and am waiting on her.

**Comment by Relequestual** [comment]
@fschiettecatte would you be able to nudge Ada on this again? Would be good to have 1.0 with a DOI for the papers!!

**Comment by fschiettecatte** [comment]
Done.

**Comment by Relequestual** [comment]
Completed with b0f490457f081a4c3da2e565ed040c661802b43d
